tests/text/EnglishPluralizerTest.php

```diff
@@ -102,9 +102,9 @@ public function testPluralForm($input, $output) {
 
 	/**
 	 * @dataProvider providerForWrongType
-	 * @expectedException \InvalidArgumentException
 	 */
 	public function testWrongTypeToPluralizeThrowsException($wrong) {
+		$this->expectException(\InvalidArgumentException::class);
```

Comment:
I think we can add the `expectExceptionMessage` to assert the expected exception message is same as result.

Reply:
Done